src/main/java/org/fiware/iam/tmforum/NotificationListener.java

```diff
@@ -0,0 +1,92 @@
+package org.fiware.iam.tmforum;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.annotation.Controller;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.fiware.iam.til.TrustedIssuersListAdapter;
+import org.fiware.iam.tmforum.product.server.api.NotificationListenersClientSideApi;
+import org.fiware.iam.tmforum.product.server.model.*;
+
+import java.util.Collection;
+import java.util.stream.Stream;
+
+@Slf4j
+@Controller("${general.basepath:/}")
+@RequiredArgsConstructor
+public class NotificationListener implements NotificationListenersClientSideApi {
+
+    private final TrustedIssuersListAdapter adapter;
+
+    private final OrganizationResolver organizationResolver;
+
+    @Override
+    public HttpResponse<EventSubscriptionVO> listenToCancelProductOrderCreateEvent(CancelProductOrderCreateEventVO data) {
+        log.warn("Received an unimplemented CancelProductOrderCreateEvent: {}", data);
```

Comment:
Why is unimplemented answered with ok? Wouldn't 501 make much more sense?

Reply:
True. Didn't think about it, since we don't give out the endpoints anyway